src/Templates/BlockTemplatesCompatibility.php

```diff
@@ -327,6 +329,11 @@ protected function get_hooks_buffer( $hooks, $position ) {
 		ob_start();
 		foreach ( $hooks as $hook => $data ) {
 			if ( $data['position'] === $position ) {
+				/**
+				 * Action to render the content of a hook.
```

Comment:
Should we remove this comment? 🤔

Reply:
It doesn't hurt: it was added, so we don't have a docblock with just a `@since` tag hanging around and throwing CS errors.